terraform/cp4ba/variables.tf

```diff
@@ -90,20 +91,14 @@ variable "ldap_admin_password" {
 variable "ldap_server" {
   description = "LDAP server "
 }
-
-# --- LDAP SETTINGS ---
 locals {
```

Comment:
Why is this a local variable?  Isn't this something that the user should be able to change?  We can default it to "cn=root"